### ISSUE TITLE
new versions on pypi

### ISSUE THREAD

**Issue by mikeatm** [yambo-code/aiida-yambo#253]
Is there a plan to put the latest versions on pypi?

**Comment by mikibonacci** [comment]
yes. the only problem up to now is the dependence on the yambopy package, which is explicit (not just a copy-paste of scripts). Yambopy is not yet on pypi, so this blocks the latest aiida-yambo versions to be available on pypi.
@palful some news on that side? maybe, we can just upload on pypi the aiida-parsing branch, and specify what branch/version it is

**Comment by chrisjsewell** [comment]
Heya, any progress on this?
This is also blocking getting a new release on conda (https://github.com/conda-forge/aiida-yambo-feedstock) and, in turn, including it in https://quantum-mobile.readthedocs.io
